swarm/network/kademlia.go

```diff
@@ -29,6 +29,16 @@ import (
 	"github.com/ethereum/go-ethereum/swarm/pot"
 )
 
+const (
```

Comment:
Any reason for creating this constants? I see no grater code clarity with them separated from NewKadParams, just an additional indirection when observing the code, and no abstraction introduced.

Reply:
I'm just allergic to literals in constructors, that's all.

Reply:
Good to know. ;) I am not allergic to unneeded indirections, but I do not like them very much.

Reply:
Too bad there are no macros in go :/